source/javascripts/pages/WorkflowsPage/components/StepBundlePanel/StepBundlePanel.tsx

```diff
@@ -0,0 +1,38 @@
+import { Box, Notification, Text } from '@bitrise/bitkit';
+import useNavigation from '../../../PipelinesPage/hooks/useNavigation';
+import { Step } from '@/models';
+
+type StepBundlePanelProps = {
+  step: Step;
+};
+
+const StepBundlePanel = (props: StepBundlePanelProps) => {
+  const { step } = props;
```

Comment:
```suggestion
const StepBundlePanel = ({ step }: StepBundlePanelProps) => {
```

Reply:
But it would be a lot cleaner to only provide it the necessary data aka the `displayName`